spec/concurrent/edge/future_spec.rb

```diff
@@ -171,6 +171,12 @@
       expect([queue.pop, queue.pop, queue.pop, queue.pop].sort).to eq [:async, :async, :sync, :sync]
     end
 
+    it 'supports setting timeout while waiting' do
+      future = Concurrent.future { sleep 1 }
```

Comment:
Can you use a shorter sleep here? We are trying to minimize our usage of sleep in the library, but since we are testing a timeout here using sleep is totally justified, but a shorter value will our test suite run slightly quicker. 😄